database/factories/PostsFactory.php

```diff
@@ -44,8 +44,8 @@
 $factory->define(Posts::class, function (Faker $faker) {
     return [
         'title'=>$faker->sentence(3, false),
-        'description'=>$faker->sentence,
-        'content'=>$faker->text,
+        'description'=>implode($faker->sentences($faker->numberBetween(1, 5))),
```

Comment:
We should use faker's method of generating only text not arrays from multiple number of sentences/paragraphs